src/Entity/FreightCollect.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/*
+ * Copyright © Eduard Sukharev
+ *
+ * For a full license, see the LICENSE file.
+ */
+
+
+namespace Ups\Entity;
+
+/**
+ * @author Eduard Sukharev <eduard.sukharev@opensoftdev.ru>
+ */
+class FreightCollect
+{
+    /**
+     * @var string
+     */
+    private $accountNumber;
+
+    /**
+     * @var Address
+     */
+    private $billReceiverAddress;
+
+    /**
+     * @return string
+     */
+    public function getAccountNumber()
+    {
+        return $this->accountNumber;
+    }
+
+    /**
+     * @param string $accountNumber
+     * @return FreightCollect
+     */
+    public function setAccountNumber($accountNumber = null)
```

Comment:
Not sure why null would be useful here. If you want to reset it to null, parameter can be supplied. I found it more upwards als once.